scripts/binary-release.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -euo pipefail
 
-branch=${BUILDKITE_BRANCH}
-commit=${BUILDKITE_COMMIT}
+branch=${BUILDKITE_BRANCH:-${GITHUB_REF##*/}}
```

Comment:
what is this magic?

Reply:
steals here: https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions
github actions doesn't give you the branch as env var